src/app/components/Loop/index.tsx

```diff
@@ -0,0 +1,425 @@
+      amount,
+      loopType,
+      destination,
+      swapRoutingFee,
+      swapFee,
+      minerFee,
+      prepayAmt,
+      channel,
+      quoteModalIsOpen,
+      advanced,
+      conf,
+      htlc,
+    } = this.state;
+    const actions: ButtonProps[] = [
+      {
+        children: (
+          <>
+            <Icon type="question-circle" theme="filled" /> {`${loopType} Quote`}
+          </>
+        ),
+        type: 'primary' as any,
+      },
+    ];
+
+    if (loopTerms === null) {
+      return null;
+    }
+
+    return (
+      <>
+        <div>
+          <Radio.Group defaultValue="a" size="large">
+            <Radio.Button value="a" onClick={this.setLoopOutType}>
+              Loop Out
+            </Radio.Button>
+            <Radio.Button value="b" onClick={this.setLoopInType}>
+              Loop In
+            </Radio.Button>
+          </Radio.Group>
+        </div>
+        <div className="Loop">
+          <InputLoopAddress
+            setLoop={this.props.setLoop}
+            setLoopIn={this.props.setLoopIn}
+            error={error}
+            initialUrl={this.props.url}
+            type={loopType}
+          />
+          {loopTerms.swap_fee_base !== '' && (
+            <Tooltip
+              overlayClassName="Loop-terms-tip"
+              title={`
+                Base fee: ${loopTerms.swap_fee_base} sats |
+                Fee rate: ${loopTerms.swap_fee_rate} sats |
+                Prepay amt: ${
+                  loopTerms.prepay_amt === undefined ? '1337' : loopTerms.prepay_amt
+                }
+                Min Swap amt: ${loopTerms.min_swap_amount} sats |
+                Max Swap amt: ${loopTerms.max_swap_amount} sats |
+                CLTV delta: ${loopTerms.cltv_delta}
+              `}
+              placement="topRight"
+              arrowPointAtCenter
+            >
+              <br />
+              <Button
+                className="Loop-form-advancedToggle"
+                onClick={this.toggleAdvanced}
+                type="primary"
+                ghost
+              >
+                {this.state.advanced === true
+                  ? 'Hide advanced fields'
+                  : 'Show advanced fields'}
+              </Button>
+            </Tooltip>
+          )}
+          <Form className="Loop" layout="vertical">
+            {url !== null && loopType === 'Loop Out' && (
+              <Form.Item>
+                <Dropdown overlay={loopOutMenu}>
+                  <a className="ant-dropdown-link" href="#">
+                    Select Channel
+                  </a>
+                </Dropdown>
+              </Form.Item>
+            )}
+            {url !== null && loopType === 'Loop In' && (
+              <Form.Item>
+                <Dropdown overlay={loopInMenu}>
+                  <a className="ant-dropdown-link" href="#">
+                    Select Channel
+                  </a>
+                </Dropdown>
+              </Form.Item>
+            )}
```

Comment:
These two behave identically, they probably shouldn't be seperate conditionals but should instead be:
```
<Dropdown overlay={loopMenu}>
```
where loopMenu is a variable that's either loopInMenu or loopOutMenu

Reply:
updated with ternary